src/main/kotlin/kweb/Element.kt

```diff
@@ -342,7 +344,7 @@ open class Element(
             return document.getElementById({}).addEventListener({}, function(event) {
                 $jsCode
             });""".trimIndent()
-        browser.callJsFunction(wrappedJS, id, eventName)
+        browser.callJsFunction(wrappedJS, JsonPrimitive(id), JsonPrimitive(eventName))
```

Comment:
I think you're right, this should have been an execute() previously rather than an evaluate(), so this is fine.